scheduler/scheduler_test.go

```diff
@@ -204,7 +212,7 @@ func TestPrepareAndExecPythonType(t *testing.T) {
 		t.Fatal(err)
 	}
 	p, r := prepareTestData()
-	pythonExecuteableName = "go"
+	pythonExecName = "go"
```

Comment:
Shouldn't this be "python"? :D

Reply:
No because then I have to install python/java in the test docker image used by circleci. I just fake it with the go executable. 😄

Reply:
Argh, I see. :D